README.md

```diff
@@ -4,7 +4,7 @@
             <picture>
                 <source srcset="https://github.com/user-attachments/assets/6d1c9fc1-11d5-47b0-bb39-a81d40d8fab0" media="(prefers-color-scheme: light)">
```

Comment:
If we are just using a single image, can we simplify this to simple markdown?

Reply:
Well I love dark mode and that's looks better on dark mode but absolutely don't mind